src/doc/unstable-book/src/language-features/asm-experimental-arch.md

```diff
@@ -164,6 +173,10 @@ This feature tracks `asm!` and `global_asm!` support for the following architect
 | CSKY         | `r31`                                   | This is the TLS register.  |
 | s390x        | `c[0-15]`                               | Reserved by the kernel. |
 | s390x        | `a[0-1]`                                | Reserved for system use. |
+| SPARC        | `r0`/`g0`                               | This is always zero and cannot be used as inputs or outputs. |
+| SPARC        | `r1`/`g1`                               | Used internally by LLVM. |
+| SPARC        | `r6`/`g6`, `r7`/`g7`                    | Reserved for system. |
+| SPARC        | `r31`/`i7`                              | Return address cannot be used as inputs or outputs. |
```

Comment:
`i6`/`o6` also need to be added to the list of stack/frame pointers.

Reply:
And g5 needs to be mentioned as reserved on sparc32.

Reply:
Good catch! I updated docs to mention them.